client/src/components/Bootstrapper.js

```diff
@@ -0,0 +1,12 @@
+import React, { useContext, useEffect } from "react";
+import { Context as AuthContext } from "contexts/AuthContext";
+
+const Bootstrapper = ({ children }) => {
+  const { refreshLoggedInUser } = useContext(AuthContext);
+  useEffect(() => {
+    refreshLoggedInUser();
+  }, []);
+  return <>{children}</>;
+};
+
```

Comment:
I like how you're using useEffect to refresh user on app loading. does this have to be its own component? would you be able to just call   `useEffect(() => {
    refreshLoggedInUser();
  }, []);` in app.js itself?